src/Section/HarvestsSection/HarvestsSection.js

```diff
@@ -23,16 +24,39 @@ class HarvestsSection extends Component {
       }
   }
 
+  getGraphData() {
+    let data = {}
+    for (var i = this.state.harvests.length - 1; i >= 0; i--) {
```

Comment:
If I reverse `this.state.harvests`, it will be changed. To perform `reverse()` it should copy `this.state.harvests`.